src/inc/corinfo.h

```diff
@@ -1686,6 +1686,8 @@ struct CORINFO_CALL_INFO
     };
 
     CORINFO_CONST_LOOKUP    instParamLookup;    // Used by Ready-to-Run
+
+    BOOL                    secureInvoke;
```

Comment:
One more nit - Could you please rename `secureInvoke` to `secureDelegateInvoke`?